src/modules/storage/LedgerStorage.ts

```diff
@@ -671,6 +729,135 @@ export class LedgerStorage extends Storages {
         });
     }
 
+    /**
```

Comment:
Please separate business logic.
We would like to call it at that point.
https://github.com/bosagora/stoa/blob/d93f1ab4f4fa24a945bc9a146ebad465e27af5fe/src/modules/storage/LedgerStorage.ts#L407-L408

Reply:
resloved